src/Components/Search/SearchBar.tsx

```diff
@@ -175,38 +181,46 @@ export class SearchBar extends Component<Props, State> {
   }
 
   renderInputComponent = inputProps => (
-    <Box>
-      <Input style={{ width: "100%" }} {...inputProps} />
-    </Box>
+    <Input style={{ width: "100%" }} {...inputProps} />
```

Comment:
cc @damassi this was where when we don't pass a custom input renderer, everything is fine. With the custom input  renderer, there's that edge case of the input ref being undefined in the autosuggest lib